azure-pipelines.yml

```diff
@@ -15,7 +15,7 @@ jobs:
  
       - script: |
           set PATH=C:\Program Files (x86)\Windows Kits\10\bin\10.0.16299.0\x64;%PATH%
-          cmake .. -G "Visual Studio 15 2017 Win64" -C../cmake/caches/Windows.cmake .. -DOMR_DDR=0
+          cmake .. -G "Visual Studio 15 2017 Win64" -C../cmake/caches/Windows.cmake .. -DOMR_DDR=0 -DOMR_TEST_COMPILER=ON -DOMR_JITBUILDER_TEST=ON
```

Comment:
Can we add these options to the Windows.cmake cache file instead?

Reply:
Let's also migrate the `-DOMR_DDR=0` as well when we make the above change.

Reply:
The problem with merging these options into Windows.cmake caches file is the Windows Jenkins builds also builds on this cache file. However, the Jenkins is using an older build system that does not support some of the Tril Test infrastructure. I added the options outside the Windows.cmake to avoid this particular problem. But I think we could create a new cmake file for Azure.

Reply:
> But I think we could create a new cmake file for Azure.

I wouldn't want to complicate things. If we can't migrate to Windows.cmake I'm ok leaving them here.